seedlet/classes/class-seedlet-custom-colors.php

```diff
@@ -293,7 +292,7 @@ function on_customize_controls_enqueue_scripts() {
 		];
 
 		wp_enqueue_script( $handle, $src, $deps );
-		wp_script_add_data( $handle, 'data', sprintf( 'var _validateWCAGColorContrastExports = %s;', wp_json_encode( $exports ) ) );
+		wp_script_add_data( $handle, 'data', sprintf( 'var seedletValidateWCAGColorContrastExports = %s;', wp_json_encode( $exports ) ) );
```

Comment:
The specific feedback here was "those variables should be prefixed". 

I'm not exactly sure what variable the reviewer is referring to, but I changed the name of the JS variable here, as well as adding a prefix to the `$handle`.

Reply:
@kjellr does this change address the feedback?

Reply:
I _think_ so, but cc @jeffikus or @danieldudzic for another set of eyes. I'm not 100% sure what they're referring to here either.

Reply:
No idea what they mean 😅

If they could provide an example, that would be helpful.

Reply:
I'll ask for clarification.

Reply:
WP Code standards are full word, camel casing with lowercase first letter. This meets that standard in my assessment.

Reply:
@jffng I got some [clarification](https://themes.trac.wordpress.org/ticket/85664#comment:8). We need to update the `_validateWCAGColorContrastExports` variable name, as well as the `validateContrastText` object name. I'm not sure that the `$handle` needs a prefix, but it seems better to give it one.

Reply:
Fixed this, mind giving this PR one more look over @kjellr ?